packages/dso-toolkit/components/Voorbeelden/10-Patronen/subtitle/subtitel.njk

```diff
@@ -0,0 +1,12 @@
+{% render '@header' %}
+
+<main>
+  <h1>Stelselcatalogus Omgevingswet</h1>
+  <p role="doc-subtitle">Subtitel</p>
```

Comment:
Dit is niet wat ik bedoelde in mijn [comment](https://github.com/dso-toolkit/dso-toolkit/pull/1291#pullrequestreview-780527354).

Ik verwacht `components/subtitle/subtitle.njk`, inclusief alles wat we daar normaal bij doen. Vervolgens wordt dat component aangeroepen:

```
<h1>Kop</h1>
{% render '@subtitle', { subtitle: 'Subtitel' } %}
```

Voor `<h1>` zullen we nooit 'n component maken, de `<h1>` zal nooit uitgebreid gaan worden. `<p role="doc-subtitle">` aan de andere kant is een composiet van m HTML element en 'n attribuut: Dat willen we vastleggen.

Reply:
Ik heb te horen gekregen dat de UX-ers niet op een losse componenten pagina staan te wachten, wat een prima verwachting is.

Neemt niet weg dat met deze opzet we geen plezier van het DSO Toolkit framework hebben: We missen reference DOM en herbruikbare componenten.

Ik heb jouw `subtitLE/subtitEL.njk` (emphasis mine) gesplitst naar `subtitel/subtitel.njk` en `_subtitle/subtitle.njk`. Nu hebben we het plezier van een herbruikbaar `@subtitle` nunjucks component en is dit component geisoleerd te "reference dommen"